CHANGES.rst

```diff
@@ -81,6 +89,26 @@ API changes
 
 - ``astropy.nddata``
 
+  - ``NDDataBase`` only defines an abstract ``uncertainty`` getter instead of
+    an actual implementation. The original was moved to ``NDData``.
+
+  - ``NDDataBase`` doesn't implement an ``uncertainty`` setter. It was moved to
+    ``NDData``. The setter was also modified:
+
+    - it wraps an ``uncertainty`` inside an ``UnknownUncertainty``
+      if no ``uncertainty_type`` attribute is present in the uncertainty instead
+      of raising an Exception.
+
+    - the uncertainty_type of the uncertainty must not be a string
```

Comment:
+ Capitalize "the"
+ Double backticks around `uncertainty_type`
+ "must not be anymore." -> "is no longer required to be, but..."